server/command.go

```diff
@@ -35,7 +35,7 @@ func (p *Plugin) getCommand() (*model.Command, error) {
 	return &model.Command{
 		Trigger:              "zoom",
 		AutoComplete:         true,
-		AutoCompleteDesc:     "Available commands: start, disconnect, help",
+		AutoCompleteDesc:     "Available commands: start, connect, disconnect, help",
```

Comment:
Connect and disconnect are available only on certain configurations. @hanzei do you think we can fill this conditionally depending on the configuration? Should we leave that to a different PR?

Reply:
Do you think it would be okay to reuse the same condition (`p.configuration.EnableOAuth && !p.configuration.AccountLevelApp`) that we have in `getAutocompleteData`? I can do it in the same PR if you think it's ok!

Reply:
I hope @mickmister will cover that in https://github.com/mattermost/mattermost-plugin-zoom/pull/249. This is how GitLab handles that case: https://github.com/mattermost/mattermost-plugin-gitlab/blob/e979912e580919b8d0b6090d1f5c02054dd53c86/server/command.go#L612-L619

Reply:
@hanzei This is a bit different as this isn't an "incomplete" setup scenario, it's really an "alternative" setup scenario.

The `connect` command may be purposely unavailable even though the plugin is in a fully-configured state. 2/5 I'm thinking this conditional check for showing the `connect` command should go in this PR.

Reply:
I'm fin with handling the `connect` case in this PR. :+1: